Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -1,11 +1,40 @@
 #include "pathStateClasses.hpp"
 
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
 bool isError; 
+Telemetry_PIGO_t commsWithTelemetry::_incomingData;
+_CruisingState_Telemetry_Return cruisingState::_returnToGround;
+_WaypointManager_Data_In cruisingState::_inputdata;
+_WaypointManager_Data_Out cruisingState::_outputdata; 
+SFOutput_t sensorFusion::sfOutputData;
+CoordinatedTurnAttitudeManagerCommands_t coordinateTurnElevation::_rollandrudder;
+AltitudeAirspeedCommands_t coordinateTurnElevation::_pitchandairspeed; 
+AttitudeData commsWithAttitude::_receivedData;
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
 
 void commsWithAttitude::execute(pathManager* pathMgr)
 {
-    //initial mode
-    pathMgr -> setState(getFromTelemetry::getInstance());
+    
+    bool newDataAvailable = GetAttitudeData(&_receivedData); // Gets attitude manager data
+
+    CoordinatedTurnAttitudeManagerCommands_t * turnCommands = coordinateTurnElevation::GetRollAndRudder();
+    AltitudeAirspeedCommands_t * altCommands = coordinateTurnElevation::GetPitchAndAirspeed();
+
+    CommandsForAM toSend {};
```

Comment:
It makes the compiler use value initialization instead of default initialization. I believe it results in the members being zero-initialized instead of being assigned random garbage